### DISCUSSION TITLE
Forking Crazy - Submission Bug

### DISCUSSION THREAD

**Discussion by tpbnick** [cssbattle/forum#4] [Bugs reports]
Chrome:
![image](https://user-images.githubusercontent.com/49327729/109981767-3984a180-7ccf-11eb-98fc-e02a6e7132a5.png)
Firefox:
![image](https://user-images.githubusercontent.com/49327729/109982114-94b69400-7ccf-11eb-860c-2e84f07ea16d.png)

I cannot submit my code for Target #8 (Forking Crazy).  I have tried multiple times in different browsers and I always get the same error from the image above.  Submissions still work on other targets.  Please help!  Thanks :smile:

**Comment by chinchang** [comment]
Hey Nick! Seems like you are submitting something strange 😅 can you please
DM us your solution on Twitter (@css_battle) so we can have a look?

**Reply by tpbnick** [reply]
Thanks for the quick reply!  I just sent you a dm.

**Reply by chinchang** [reply]
Just to update here: this is getting fixed

**Comment by chinchang** [comment]
FIXED now!
